extensions/wikia/WikiFactory/SpecialWikiFactory_ajax.php

```diff
@@ -570,16 +568,15 @@ function axWFactorySaveVariable() {
 				 * catch parse errors
 				 */
 				ob_start();
-				if( eval( $tEval ) === FALSE ) {
+				if ( eval( $tEval ) === FALSE ) {
 					$error++;
 					$return = Wikia::errormsg( "Syntax error, value is not valid PHP structure. Variable not saved." );
-				}
-				else {
+				} else {
 					$cv_value = $__var_value;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$__var_value" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)